openmc/geometry.py

```diff
@@ -73,19 +73,16 @@ def export_to_xml(self, path='geometry.xml'):
             Path to file to write. Defaults to 'geometry.xml'.
 
         """
-        # Clear OpenMC written IDs used to optimize XML generation
-        openmc.universe.WRITTEN_IDS = {}
```

Comment:
If I recall correctly, @nhorelik introduced this to greatly accelerate the exportation of large geometries. Have you tried exporting BEAVRS with this change? Is it any slower?